src/contraction/src/contractGraph_driver.cpp

```diff
@@ -95,20 +116,27 @@ do_pgr_contractGraph(
 #ifdef DEBUG
             undigraph.print_graph(log);
 #endif
-            //pgr_dijkstra(undigraph, paths, start_vid, end_vertices, false);
-        }
 
-        size_t count(count_tuples(paths));
 
-        if (count == 0) {
-            (*return_tuples) = NULL;
-            (*return_count) = 0;
-            log << 
-                "No paths found between Starting and any of the Ending vertices\n";
-            *err_msg = strdup(log.str().c_str());
-            return;
+            /*
+            Function call to get the contracted graph.
+
+        return_tuples = get_memory (1, pgr_contracted_blob);
+            fetch_contracted_graph(undigraph,level,
+              contracted_graph_name,contracted_graph_blob,removedEdges,
+              removedVertices,psuedoEStream
+            );
+        (return_tuples)->contracted_graph_name=strdup(contracted_graph_name.str().c_str());
```

Comment:
This goes out of the if then else, and of course only once.
only the call to undirected goes in the if